docs/src/pages/en/in-the-wild.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Media Chrome in the wild
+description: Links to other resources for all things Media Chrome and the ideas behind Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+<h2>Using Media Chrome</h2>
```

Comment:
long term, I think something like https://babeljs.io/en/users.html would be cool.